FabricExample/android/gradle.properties

```diff
@@ -39,3 +39,5 @@ newArchEnabled=true
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
 hermesEnabled=true
+
+isScreensExampleApp=true
```

Comment:
We should add this flag to all of our example apps, so that it means what the name suggests.

Reply:
Changed in https://github.com/software-mansion/react-native-screens/pull/2168/commits/bd65c0c7bb6d6a12facbf79e40c1a04b89aebb4f